iam-login-service/src/main/webapp/WEB-INF/views/iam/login.jsp

```diff
@@ -72,62 +72,129 @@
             <h3>Welcome to <strong>${iamOrganisationName}</strong>
       </h3>
         </div>
+
+        <c:if
+            test="${
+                (
+                    loginPageConfiguration.localAuthenticationVisible
+                    or param.sll != null
```

Comment:
Can `or` be `||` (for uniformity)?

Reply:
Sure..Updated the code to be consistent,